app/views/alchemy/admin/elements/_element.html.erb

```diff
@@ -77,3 +94,12 @@
     </div>
   <% end %>
 <% end %>
+
+<!-- <script type='application/javascript'>
+  $(function() {
+    $(document).on('click', '.content-group-header', function() {
+      $(this).parent().toggleClass('expanded');
+      return false;
+    });
+  })
+</script> -->
```

Comment:
This needs to go into the element editors js file.

Reply:
@tvdeyen yes, it is.  I just forgot to remove it here.